plank/services/sample2.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019-2021 VMware, Inc.
```

Comment:
Lots more comments about the intent of this and what is going on.

Reply:
i'm going to refactor this service to do something more meaningful

Reply:
nice.

Reply:
@daveshanley i just added a new service that replaces the sample2 service. along with that i realized that there are a couple critical features missing in the current stompserver package so i went ahead and implemented them along with some tests. it would be great if you can provided detailed feedback on the following:

- New service
  - https://github.com/vmware/transport-go/pull/15/commits/747353cdf76237e768d854915c710ca9d1f9238e
- STOMP server changes
  - https://github.com/vmware/transport-go/pull/15/commits/c6ba881543adffaff5fb5f470602b17c105bf430
  - https://github.com/vmware/transport-go/pull/15/commits/d8a5bfbe01a9752fa71a806b61568157a9bf73c8